### DISCUSSION TITLE
How to use useInfiniteScroll on a component with the scroll of "body"?

### DISCUSSION THREAD

**Discussion by onurusluca** [vueuse/vueuse#1581] [General]
Hi! 

I want to use the [useInfiniteScroll](https://vueuse.org/core/useinfinitescroll/#useinfinitescroll) but I don't know where to add the `ref="el"`. Because the scroll is for the body and not the component I want to add the scroll. Basically I want to add the useInfiniteScroll to the page and not to an element in a component.

**Comment by ar-rohman** [comment]
You can directly assign `el` to `document` without adding ref in your vue template.
```js
const el = ref(document);
```
